Dockerfile

```diff
@@ -1,50 +1,41 @@
-FROM python:3.5
-MAINTAINER Pit Kleyersburg <pitkley@googlemail.com>
```

Comment:
Shouldn't we leave in the maintainer label? (or upgrade the line according to https://docs.docker.com/engine/reference/builder/#maintainer-deprecated )
I like having a point of contact in the dockerfile itself.

Reply:
I personally don't mind the maintainer gone, but if we keep it, then we should definitely use the `LABEL` instruction.

If we keep it, we should also maybe update it to reference this repository rather than a single person, since this never has been only my effort, but rather that of multiple great people.

*If* you add the `LABEL`, please add it as the last instruction in the `Dockerfile`, such that a update of the label will not require the whole image to be rebuilt.